Sources/llbuild2fx/FunctionInterface.swift

```diff
@@ -9,7 +9,7 @@
 import NIOConcurrencyHelpers
 import NIOCore
 
-public final class FXFunctionInterface<K: FXKey> {
+public final class FXFunctionInterface<K: FXKey>: @unchecked Sendable {
```

Comment:
why `@unchecked Sendable`? If you use `NIOLockedValueBox` instead of `NIOLock` you can make it `Sendable` without having to turn off all concurrency checking